docs/src/main/resources/microsite/data/commons.yml

```diff
@@ -0,0 +1,7 @@
+repoUrl: "https://github.com/47deg/fetch"
+organizationUrl: "https://www.47deg.com"
+organization: "47 Degrees"
+description: "Simple & Efficient data for Scala"
```

Comment:
:thinking:

Reply:
That is the description it appears in the new redesign. I´ll make sure it will be that way.

Reply:
Has this been checked?  I mean, that description meaning is a bit... empty? 🤷